src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/Startup.cs

```diff
@@ -92,12 +92,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
             app.UseIdentityServer();
 #endif
-
-            app.UseMvc(routes =>
+            app.UseEndpoints(endpoints =>
```

Comment:
This should also change to be similar to the other templates in `ConfigureServices`. This should have `AddControllersWithViews()`, and also `AddRazorPages()` when Identity is present.